### ISSUE TITLE
Casting with usage dice

### ISSUE THREAD

**Issue by pandres** [Vladar4/itdr#3]
Would it be too disruptive to implement magic casting with a dice usage the same way as the warrior has a damage die?

**Comment by GAZ082** [comment]
@pandres Mutations? Were that in the Zweihander book I sold you some time ago? Can't believe I found you here. ;)

**Comment by pandres** [comment]
@GAZ082 more like "Mutants of Ixx", but they are not fully compatible. I really like the system and I'm looking for ways of doing the weird things players usually ask for. I don't remember who sold me the Zwei original, lol.

**Comment by GAZ082** [comment]
@pandres I did! The moment I checked your profile and was Cordoba, I knew it was you. LOL. I'm also interested in this little system. Anyways, I wont spam anymore the thread. :P

**Comment by Vladar4** [comment]
> Would it be too disruptive to implement magic casting with a dice usage the same way as the warrior has a damage die?

Depends on the exact wording. Casting could be done numerous ways, just keep its level progression and damage output roughly balanced with other features.

For the Features present in the book I tried not to add any new "resource" fields for the player to track. That's why Mystics drain their own HP to cast, and Thaumaturges roll Gift dice to check if they are out of juice. For your own homebrew Features you can forgo that limitation, of course.

**Comment by pandres** [comment]
> I tried not to add any new "resource" fields for the player to track.

Good point, this keeps the spirit of the original dnd, I'll take it in account.
